include/aca_arp_responder.h

```diff
@@ -108,11 +108,11 @@ class ACA_ARP_Responder {
   int delete_arp_entry(arp_config *arp_config_in);
 
   /* Data plane Ops */
-  int arp_recv(uint32_t in_port, void *vlanmsg, void *message);
-  void arp_xmit(uint32_t in_port, void *vlanmsg, void *message, int is_find);
+  int arp_recv(uint32_t in_port, void *vlanmsg, void *message, int of_connection_id);
```

Comment:
Please see if for merge request of OF 1.3 (master branch current choice), this is still needed?
Or it is just for OF 1.0 cbench?

Reply:
fixed